builders/ethereum/dev-env/hardhat.md

```diff
@@ -52,13 +52,29 @@ You will need to create a Hardhat project if you don't already have one. You can
 4. Create a Hardhat project
 
     ```sh
-    npx hardhat init
+    npx hardhat --init
     ```
 
     !!! note
-        `npx` is used to run executables installed locally in your project. Although Hardhat can be installed globally, it is recommended to install it locally in each project so that you can control the version on a project-by-project basis.
+        `npx` is used to run executables installed locally in your project. Although Hardhat can be installed globally, installing it locally in each project is recommended so you can control the version on a project-by-project basis.
 
-5. A menu will appear, which will allow you to create a new project or use a sample project. For this example, you can choose **Create an empty hardhat.config.js**, which will create a Hardhat configuration file for your project
+
+5. You'll be prompted with a series of questions to set up your project:
```

Comment:
This seems to have changed a bit. 
![hardhat-prompts01](https://github.com/user-attachments/assets/9676d764-cca6-4089-ba7b-74015105b08c)

Reply:
Ah that is the old version of Hardhat